src/components/PurchaseOrder/LineListing/LineListing.js

```diff
@@ -22,15 +22,18 @@ class LineListing extends Component {
     const url = match.url;
 
     history.push(`${url}/po-line/view/${meta.id}`);
-  }
+  };
 
   render() {
     const { poLines } = this.props;
     const resultsFormatter = {
       'poLineNumber': item => toString(get(item, ['poLineNumber'], '')),
-      'acquisitionMethod': item => toString(get(item, ['acquisitionMethod'], '')),
-      'owner': item => toString(get(item, ['owner'], '')),
-      'poLineDescription': item => toString(get(item, ['poLineDescription'], '')),
+      'title': item => toString(get(item, 'title', '')),
+      'productId': item => toString(get(item, 'details.productIds', [])
+        .reduce((acc, { productId }) => `${acc}${acc.length ? ',' : ''} ${productId}`, '')),
```

Comment:
Could you please describe what these lines do? it's not clear from the code

Reply:
this code returns list of productIds from array 
 ```
productIds = [{
productId: <string>,
productIdType: <string>
}, ...]
```

Reply:
I don't see any `productIdType` using, only `productId`.
I think it could be smaller and cleaner, something like 
`_.map(get(item, 'details.productIds', []), 'productId').join(, )`

Reply:
thanks, great solution

Reply:
Fixed.